src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -1,41 +1,27 @@
 <template>
-<div id="conditionArea"
+<div v-if="condition"
      class="c-cs-editui__conditions"
-     :class="['widget-condition', { 'widget-condition--current': isCurrent }]"
+     :class="['widget-condition', { 'widget-condition--current': currentConditionIdentifier && (currentConditionIdentifier.key === conditionIdentifier.key) }]"
 >
     <div class="title-bar">
         <span
             class="c-c__menu-hamburger"
-            :class="{ 'is-enabled': !isDefault }"
-            @click="expanded = !expanded"
+            :class="{ 'is-enabled': !condition.isDefault }"
+            @click="expanded = !condition.expanded"
         ></span>
         <span
             class="is-enabled flex-elem"
             :class="['c-c__disclosure-triangle', { 'c-c__disclosure-triangle--expanded': expanded }]"
-            @click="expanded = !expanded"
+            @click="expanded = !condition.expanded"
         ></span>
         <div class="condition-summary">
-            <span v-if="isDefault"
-                  class="condition-name"
-            >Default
-            </span>
-            <span v-else
-                  class="condition-name"
-            >[condition name]
-            </span>
-            <span v-if="isDefault"
-                  class="condition-description"
-            >When all else fails
-            </span>
-            <span v-else
-                  class="condition-description"
-            >[condition description]
-            </span>
+            <span class="condition-name">{{ condition.definition.name }}</span>
+            <span class="condition-description">{{ condition.definition.name }}</span>
```

Comment:
condition description is displaying `condition.definition.name` - probably should be something else?

Reply:
Added TODO: description should be derived from criteria